resources/src/config.php

```diff
@@ -21,7 +21,7 @@ public function __construct()
                 ),
                 // Paths of all the required folders
                 'paths' => array(
-                    "resources" => "/resources",
+                    "resources" => "/../resources",
                     "images" =>  "/asset/images",
                     "css" =>  "/asset/css",
                     "js" => "/asset/js"
```

Comment:
So is it now pointing to folder inside public-html?

Reply:
resources is **outside** public-html, hence `/../`. And  `asset` is inside public-html